app/feature/feature-odyssey/src/main/kotlin/com/hedvig/android/feature/odyssey/step/singleitemcheckout/SingleItemCheckoutDestination.kt

```diff
@@ -89,56 +105,175 @@ private fun SingleItemCheckoutScreen(
   navigateUp: () -> Unit,
   closeClaimFlow: () -> Unit,
 ) {
+  var bottomSheetText by remember { mutableStateOf<Pair<String, String>?>(null) }
+  val explanationSheetState = rememberModalBottomSheetState(true)
+  if (bottomSheetText != null) {
+    SingleItemCheckoutInfoBottomSheet(
+      onDismiss = { bottomSheetText = null },
+      sheetState = explanationSheetState,
+      explanationTitleAndText = bottomSheetText ?: Pair("", ""), // todo: do I like it here?
```

Comment:
You can write it like this
```kotlin
  var bottomSheetText by remember { mutableStateOf<Pair<String, String>?>(null) }
  val explanationSheetState = rememberModalBottomSheetState(true)
  val bottomSheetTextValue = bottomSheetText
  if (bottomSheetTextValue != null) {
    SingleItemCheckoutInfoBottomSheet(
      onDismiss = { bottomSheetText = null },
      sheetState = explanationSheetState,
      explanationTitleAndText = bottomSheetTextValue,
    )
  }
```
And this way you will get the right null safe check in there